docs/fastdds/env_vars/env_vars.rst

```diff
@@ -2,3 +2,12 @@
 
 Environment variables
 =====================
+
+This is the list of all environment variables that affect the behavior of *Fast DDS*:
```

Comment:
It doesn't seem right to say it's "the list of all the..." and then only have one.